test/integration.js

```diff
@@ -26,6 +26,21 @@ describe('c8', () => {
     output.toString('utf8').should.matchSnapshot()
   })
 
+  it('supports exeternally set NODE_V8_COVERAGE', () => {
+    const { output } = spawnSync(nodePath, [
+      c8Path,
+      '--exclude="test/*.js"',
+      '--clean=false',
```

Comment:
> I think the test suite still has some isolation problem.

Use `--temp-directory` option here to write coverage reports to a separate directory.

Reply:
@shinnn the goal of this test is indeed to use the `env` and not `--temp-directory` to set the path. Look at line 38!

Reply:
Ah, indeed.